exe/cli/cmd/ffs_push.go

```diff
@@ -1,23 +1,31 @@
 package cmd
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
 	"errors"
+	"io"
+	"os"
 	"time"
 
 	"github.com/caarlos0/spin"
 	"github.com/ipfs/go-cid"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"github.com/textileio/powergate/api/client"
+	"github.com/textileio/powergate/ffs"
 )
 
 func init() {
-	ffsAddCmd.Flags().StringP("token", "t", "", "FFS access token")
+	ffsPushCmd.Flags().StringP("token", "t", "", "FFS access token")
+	ffsPushCmd.Flags().StringP("config", "c", "", "Optional path to a file containing cid storage config json, optional stdin otherwise")
```

Comment:
Nit: maybe change the second use of _optional ..._ to _defaults to stdin otherwise_.

Reply:
Yea, this is a tough one to describe because stdin also is optional. Maybe "falls back to stdin otherwise", but if no config is provided, it just uses the default configured for the API.